### DISCUSSION TITLE
How to make one project use one Node version while running another project on another Node version at the same time?

### DISCUSSION THREAD

**Discussion by ashkan-ahmadi** [coreybutler/nvm-windows#922] [General] [answered]
Hi

I have 2 projects that need to run at the same time, one on Node v16.18.1 and another on v14.2.0. I have followed [How can I use two different versions of Node.js at the same time?](https://github.com/coreybutler/nvm-windows/wiki/Common-Issues#how-can-i-use-two-different-versions-of-nodejs-at-the-same-time) and it's working fine for checking `npm --version` but how can I run `npm run dev` on v16.18.1 on project1 while running `npm run dev` on v14.2.0 on project2?

Thanks

**Comment by coreybutler** [comment, marked as answer]
**Update**
I've updated the examples in the [How can I use two different versions of Node.js at the same time?](https://github.com/coreybutler/nvm-windows/wiki/Common-Issues#how-can-i-use-two-different-versions-of-nodejs-at-the-same-time).

You should be able to run it the same way you'd display the version.

Assuming a `package.json` file like this:

```js
{
  "name": "tmp",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "dev": "echo \"running script from package.json...\""
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "porthog": "^1.0.1"
  }
}
```

To execute the `dev` script, run a command like this:

<img src="https://user-images.githubusercontent.com/770982/215180807-ba0f273c-a9d2-4f4b-83ba-9396f1aa0217.png" width="550"/>

**Reply by ashkan-ahmadi** [reply]
Amazing. The `C:\tmp` part on the screenshot clarified it, I believe. Then if I'm not wrong, I need to run `C:\project1>C:\Users\.....\v16.18.1\npm run dev` in one terminal and then run `C:\project2>C:\Users\.....\v14.2.0\npm run dev` in another terminal, correct?

**Reply by coreybutler** [reply]
You can run them in separate terminals if you like, but it's not required unless you're trying to run both processes at the same time.

**Reply by ashkan-ahmadi** [reply]
Amazing thank you very much. I just tested it and it worked flawlessly.
